apps/web/src/server/applications/case/representations/application-representations.view-model.js

```diff
@@ -57,16 +58,18 @@ const getRedacted = ({ redacted }) => (redacted ? 'Redacted' : 'Unredacted');
  *
  * @param {object} args
  * @param {Array.<{reference: string, organisationName?: string, firstName?: string , lastName?: string, received: string, redacted: boolean, status: string, id: string}>} args.items
+ * @param {string} caseId
  * @returns {Array.<{}>}
  */
-export function getRepresentationsViewModel({ items }) {
+export function getRepresentationsViewModel({ items }, caseId) {
 	return items.map((rep) => ({
 		reference: rep.reference,
 		title: getTitle(rep),
 		received: formatDate(rep),
 		redacted: getRedacted(rep),
 		status: getStatus(rep),
-		id: rep.id
+		id: rep.id,
+		link: getFinalRepPageUrl(rep, caseId, 'summary')
```

Comment:
summary is a magic string ? is there a shared place to put this ? 

If not just mark as resolved.